README.md

```diff
@@ -2,14 +2,24 @@
 
 ## Getting started
 
-- Download [Quarto](https://quarto.org/)
-- Clone this repository
-- In the root
-``` bash
-quarto render
-```
+### Setup
+
+- Install Nix using [nix-installer](https://zero-to-nix.com/start/install). The
+  official installer should also work, but may require additional configuration
+  that is not covered here.
```

Comment:
```suggestion
- Install Nix using [nix-installer](https://zero-to-nix.com/start/install).
```

I'd just refer to installing  nix and avoid (a) installati may fail, (b) additional unspecified config may be required.

Reply:
:+1: